### ISSUE TITLE
Wago URL copyable in "Receiving display information" window

### ISSUE THREAD

**Issue by Nnoggie** [WeakAuras/WeakAuras2#4890]
**Is your feature request related to a problem? Please describe.**
When getting linked a WeakAura ingame it often gets stuck sending. As a receiver i then often ask for the wago url instead.

**Describe the solution you'd like**
If a WeakAura has it, the Wago url could be embedded into the clickable link and displayed in the receiving window.
![image](https://github.com/WeakAuras/WeakAuras2/assets/27819512/1613f936-9332-4e06-b33c-27546824889a)

**Describe alternatives you've considered**
Fix the sending to never fail but thats a rough one.

**Comment by InfusOnWoW** [comment]
I've never heard about problems sharing auras, so that would be something to investigate.

But this also feels like the wrong solution, we can't easily make text in the tooltip selectable/copyable, so the user would have to copy that by hand. That sounds like something no one, except you, would use. 

Imho, if we think having the url is worthwhile, then adding it to the text we send via chat would be much more useful. That is we would insert [WA-NAME] ( URL ) into the chat box. Chat Addons, e.g. Prat, elvui and BasicChatMods, already have url detection and make those clickable.

**Comment by Nnoggie** [comment]
The sending often fails on big auras, maybe this has something to do with the 10.2 changes to Addon Messages? (i did not investigate if WA has adjusted to this though).
I think the suggested approach of having the URL in the chat is also a good one 👍

**Comment by mrbuds** [comment]
> i did not investigate if WA has adjusted to this though

WA use AceComm which use ChatThrottleLib which last update was a toc update on 15 Oct 2014

**Comment by Nnoggie** [comment]
10.2 added throttle restrictions for PARTY and RAID AddOn Messages
10 messages burst, 1 message per second all per prefix.

Using the default CTL defaults will break. This can be fixed by manually changing the throttle values when sending over PART or RAID.
A better solution that respects the new rules should be implemented in CTL upstream i think, but until that happens the following fix works for MDT and should fix the problem here aswell.

https://github.com/Nnoggie/MythicDungeonTools/blob/8153df21ae50d35d1231a97c55a17708421403aa/Modules/Transmission.lua#L838

**Comment by InfusOnWoW** [comment]
@Nevcairiel can you take a look at that? 

Nnoggie says that these defaults:
ChatThrottleLib.MAX_CPS = 255
ChatThrottleLib.BURST = 2550

would be correct in 10.2.

**Comment by mrbuds** [comment]
I think both WHISPER & GUILD channels still work fine with old values

**Comment by Nevcairiel** [comment]
As far as I understand, the new limits are per prefix, not global. Hence just changing the values is not actually properly correct. And the limits also depend on channels and circumstances.

If someone wants to implement the full limitations, feel free to make a PR against https://github.com/WoWUIDev/Ace3, and once merged I'll also update the standalone CTL.

**Comment by Nnoggie** [comment]
> I think both WHISPER & GUILD channels still work fine with old values

This is correct. Only PARTY and RAID channels are affected by the new restrictions. And yes, just changing the values is not the correct approach, but atleast communications don't get dropped anymore.

**Comment by InfusOnWoW** [comment]
I'll take a look at LibChatThrottleLib, though it seems the first task would be to figure out what the actual limits are and whether they are per channel. Though no promises on actually creating a PR.
